src/main/index.ts

```diff
@@ -271,30 +271,57 @@ app.on('web-contents-created', (_ev, contents) => {
     contents.session.storagePath &&
     contents.session.storagePath.indexOf('webxdc_') !== -1
   if (is_webxdc) {
-    contents.on('will-navigate', (e, navigationUrl) => {
+    const webxdc_open_url = (url: string) => {
+      if (url.startsWith('mailto:') || url.startsWith('openpgp4fpr:')) {
+        // handle mailto in dc
```

Comment:
With the danger of being new here and asking annoying questions: What does this comment exactly imply? I read it as "Handle mailto: in DeltaChat", but exactly which parts of it? Where would we handle it instead if not in DeltaChat?

Reply:
> Where would we handle it instead if not in DeltaChat?

in the system handler for mailto, which can be deltachat, but is likely a traditional email MUA like thunderbird